src/main/resources/de/tum/cit/ase/ares/api/templates/phobos/phobos-common.sh

```
@@ -53,10 +129,21 @@ parse_cfg_policy() {
   local ro="${tdir}/ro.paths" rw="${tdir}/rw.paths" hide="${tdir}/hide.paths" net="${tdir}/net.rules"
   : >"$ro"; : >"$rw"; : >"$hide"; : >"$net"
   local sec=""
-  while IFS= read -r line || [[ -n "$line" ]]; do
-    line="${line%%#*}"; line="$(echo "$line" | sed -E 's/^[[:space:]]+//; s/[[:space:]]+$//')"
+  local raw line name status
+  # Section-local, so it never carries a count from another section or file.
+  local timeout_values=0
+  # Per-file, so the caller can tell "this policy set no timeout" from "this
+  # policy set a timeout of zero".
+  PARSED_TIMEOUT_SET=0
```

Shellcheck (0.11.0)

[warning] PARSED_TIMEOUT_SET appears unused. Verify use (or export if used externally).

(SC2034)